test/dotnet-grpc.Tests/CommandBaseTests.cs

```diff
@@ -0,0 +1,367 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Grpc.Dotnet.Cli.Commands;
+using Grpc.Dotnet.Cli.Internal;
+using Grpc.Dotnet.Cli.Options;
+using Microsoft.Build.Definition;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Locator;
+using NUnit.Framework;
+
+namespace Grpc.Dotnet.Cli.Tests
+{
+    [TestFixture]
+    public class BindMethodFinderTests
+    {
+        private static readonly string ProtoUrl = "https://raw.githubusercontent.com/grpc/grpc-dotnet/edf4b478e5d35f19e69943eb807a99709fc8de3b/examples/Proto/greet.proto";
```

Comment:
I should remove the part where tests fetch content from the web, otherwise it's going to be a reliability nightmare.